### PR TITLE
blog: Add GSoC'24 UEFI GOP, Part III Post

### PR DESCRIPTION
**Pull request by procub3r**

### CODE REVIEW

**Comment by procub3r** [comment]
Fixed all linter issues.

File: content/blog/2024-08-07-gsoc-uefi-gop.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+### Generic console device interface
+
+A generic console device interface was proposed by [PR #1464](https://github.com/unikraft/unikraft/pull/1464) submitted by [thass0](https://github.com/thass0) on GitHub. This interface, exposed through a library called `ukconsole`, allows you to:
+
+- Register console devices, each of which uses its own underlying interfaces (serial, VGA, UEFI GOP etc).
+- Query registered console devices.
+- Write bytes to either or all registered console devices to print logs.
+- Read bytes from a given console device to take user input.
+
+All of my work has been rebased onto the work done in `PR #1464`.
```
Comment (StefanJum) [inline content/blog/2024-08-07-gsoc-uefi-gop.mdx:65]: link the pr

**Review by StefanJum — CHANGES_REQUESTED** [review]
@procub3r please use your full name for the signoff, commiter and author fields.

```
Author: procub3r <procub3r@proton.me>
Commit: procub3r <procub3r@proton.me>

    blog: Add GSoC'24 UEFI GOP, Part III Post
    
    Signed-off-by: procub3r <procub3r@proton.me>
```

Use `git config --global user.name` and then amend the commit.

**Comment by StefanJum** [comment]
Also @mogasergiu please take a quick look.

**Review by mogasergiu — APPROVED** [review]
Reviewed-by: Sergiu Moga <sergiu@unikraft.io>

File: content/blog/2024-08-07-gsoc-uefi-gop.mdx (hunk 1 above)
Comment (procub3r) [reply content/blog/2024-08-07-gsoc-uefi-gop.mdx]: linked :+1:

**Comment by StefanJum** [comment]
@procub3r the author name and email is still wrong
```
Author: procub3r <procub3r@proton.me>
Commit: Sriprad Potukuchi <sripradp@proton.me>

    blog: Add GSoC'24 UEFI GOP, Part III Post
    
    Signed-off-by: Sriprad Potukuchi <sripradp@proton.me>
```

**Comment by procub3r** [comment]
@StefanJum Fixed now. Didn't know I had to change that separately :sweat_smile:

**Review by StefanJum — APPROVED** [review]
Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>
Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Pull request merged**